versions.tf

```
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.74.0"
+      version = ">= 5.75.0"
```

```suggestion
      version = ">= 5.77.0"
```

Due to https://github.com/hashicorp/terraform-provider-aws/pull/40168